packages/flame/lib/src/components/position_component.dart

```diff
@@ -121,14 +138,27 @@ abstract class PositionComponent extends BaseComponent {
   /// Get the absolute center of the component
   Vector2 get absoluteCenter => absoluteParentPosition + center;
 
-  /// Angle (with respect to the x-axis) this component should be rendered with.
-  /// It is rotated around its anchor.
-  double angle;
+  /// Flip the component horizontally around its anchor point.
+  void flipHorizontally() => _transform.flipHorizontally();
 
-  /// Anchor point for this component. This is where flame "grabs it".
-  /// The [position] is relative to this point inside the component.
-  /// The [angle] is rotated around this point.
-  Anchor anchor;
+  /// Flip the component vertically around its anchor point.
+  void flipVertically() => _transform.flipVertically();
+
+  /// Flip the component horizontally around its center line.
+  void flipHorizontallyAroundCenter() {
```

Comment:
Will these work if they are called two times in a row? To first flip it, and then flip it back?

Reply:
I hope so. Probably can add a test for this.